lib/fpm/package/deb.rb

```diff
@@ -83,6 +83,9 @@ class FPM::Package::Deb < FPM::Package
 
   option "--group", "GROUP", "The group owner of files in this package", :default => 'root'
 
+  option "--mode", "MODE", "The mode modification to make to the files in this " \
+    "package. Use the same format as passing the mode tag into tar", :default => 'NOCHANGE'
```

Comment:
Does this need a default value? If not specified in this definition (remove `:default => ...`), the default is `nil`. Unless ... "NOCHANGE" has some special meaning to tar?